udsoncan/connections.py

```diff
@@ -735,15 +735,24 @@ def __init__(self, windll: str, rxid: int, txid: int, name: Optional[str] = None
         txFlags = TxStatusFlag.ISO15765_CAN_ID_29.value if txid >> 11 else TxStatusFlag.ISO15765_CAN_ID_11.value
 
         # Set up a J2534 interface using the DLL provided
-        self.interface = J2534(windll=windll, rxid=rxid, txid=txid, txFlags=txFlags)
+        try:
+            self.interface = J2534(windll=windll, rxid=rxid, txid=txid, txFlags=txFlags)

```

Comment:
You changed an exception for another. What is the purpose?
The new exception doesn't say what was looked for, can you add it?

Reply:
More information about inaccessible dll file.
In some times error message are different and we should control these.
In future we need create common udsoncan exceptions for common behaviour when dll or device inaccessible

Reply:
And we should controls in annotations which exceptions can be raised this library. We have one type RuntimeException for all situations… more stable application when it  handles much more exception